index.html

```diff
@@ -54,7 +54,174 @@ <h1>I’m Godfrey
     </div>
   
   </section>
+  <section class="work">
+    <div class="container_work">
+      <article id="pic1" class="work_content">
+            <img src="./image/Snapshoot Portfolio (1).svg" alt="">
+          <div class="work_description">
+            <h2>Tonic</h2>
+            <ul class="conduct">
+              <li class="conduct_head">CANOPY</li>
+              <li><img src="./image/Counter.svg" alt=""></li>
+              <li class="conduct_para">Back End Dev</li>
+              <li><img src="./image/Counter.svg" alt=""></li>
+              <li class="conduct_year">2015</li>
+            </ul>
+            <p class="work_paragraph">A daily selection of privately personalized reads; 
+              no accounts or sign-ups required.</p>
+            <ul class="work_languages">
+             <li>html</li>
+             <li>css</li>
+             <li>javascript</li>
+            </ul>
+              <button class="btn">See Project</button>
```

Comment:
- [ ] Great job using the button tag correctly here, but I think there is a project requirement you missed, 😄  You didn't implement the hover and touch state of the button. Please consider implementing this from the design as its part of the requirement for this milestone: 👋 
> Implement the button interactions (enable, hover, etc.).